lazy-pages/src/sys/unix.rs

```diff
@@ -59,7 +65,7 @@ cfg_if! {
     } else if #[cfg(all(target_os = "macos", target_arch = "aarch64"))] {
         unsafe fn ucontext_get_write(ucontext: *mut nix::libc::ucontext_t) -> Option<bool> {
             // See https://developer.arm.com/documentation/ddi0595/2021-03/AArch64-Registers/ESR-EL1--Exception-Syndrome-Register--EL1-
-            const WNR_BIT_MASK: u32 = 0b0100_0000; // Write not Read bit
+            const WNR_BIT_MASK: u32 = 0b100_0000; // Write not Read bit
```

Comment:
@ark0f  please help with this bit)) cannot find a changes in master for this bit.

Reply:
A ok see now